protocols/gossipsub/src/behaviour.rs

```diff
@@ -3702,20 +3640,32 @@ mod local_test {
         }
     }
 
-    impl Arbitrary for Rpc {
+    impl Arbitrary for proto::RPC {
         fn arbitrary(g: &mut Gen) -> Self {
-            let mut rpc = empty_rpc();
-
-            for _ in 0..g.gen_range(0..10u8) {
-                rpc.subscriptions.push(test_subscription());
-            }
-            for _ in 0..g.gen_range(0..10u8) {
-                rpc.messages.push(test_message());
-            }
-            for _ in 0..g.gen_range(0..10u8) {
-                rpc.control_msgs.push(test_control());
+            match u8::arbitrary(g) % 3 {
```

Comment:
we only send one type of gossipsub message so I'd say this reflects better on our usage